index.html

```diff
@@ -69,14 +69,45 @@ <h2>Tonic</h2>
                     <li>CSS</li>
                     <li>JavaScript</li>
                 </ul>
-                <button type="button" class="btn">See Project</button>
+                <button type="button"  data-modal-target="#modal" class="btn" title="0">See Project</button>
+            <div class="modal" id="modal">
+                <div class="modal-header">
+                    <div class="title"></div>
+                    <buttton data-close-button class="close-button">&times;</buttton>
+                </div>
+                <div class="contentt">
+                    <div class="footer">
+                        <span></span>
+                        <img src="./image/Counter.png" alt="">
+                        <span></span>
+                        <img src="./image/Counter.png" alt="">
+                        <span></span>
+                    </div>
+                </div>
+                <div class="modal-body">
+                    <img src="" alt="">
+            
+                    <div class="grid-container">
+                        <div class="grid-lorem">
+                        </div>
+                        <div class="grid-button">
+                            <ul class="languagessss"></ul>
+                            <hr>
+                            <div class="button-container">
+                                <button class="live">See Live<i class="fa-solid fa-heart-pulse"></i></button>
+                                <button class="live">See Source<i class="fa-brands fa-github"></i></button>
```

Comment:
- [ ]  Your detail popup window `See Live` and `See Source` button is not working when I click the button that why you should add the actual source code and demo link for that buttons .If you don't have another project you can add portfolio demo link and source code link for all the project as temporarily.